web/src/app/shared/models/index.model.ts

```diff
@@ -3,6 +3,7 @@ export * from './model.model';
 export * from './activity.model';
 export * from './contributor.model';
 export * from './event.model';
+export * from './issue.model';
```

Comment:
I don't see an `issue.model` file?

Reply:
Actually its not part of this PR . `issue.model` was missing from `index.model.ts` file. So I have added here

Reply:
Oh ok, nice spot, thank you 👍

Reply:
:)